test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -2448,7 +2448,9 @@ public void run() {
             final Process p = new ProcessBuilder(childArgs).start();
             long start = System.nanoTime();
 
-            p.waitFor(10, TimeUnit.MILLISECONDS);
+            if (p.waitFor(10, TimeUnit.MILLISECONDS)) {
+                System.out.println("WaitFor didn't wait long enough: " + (System.nanoTime() - start));
```

Comment:
Either the condition or the message seems wrong here. If waitFor returns true then the process has exited and we obviously did wait long enough.

Reply:
This code is diagnostic.  
After switching to native sleep, I had intermittent failures claiming it did not sleep long enough.
I was unable to find a specific cause for those failures.
Many of the tests fail to check if the sleep processes terminate prematurely and if the executable is not found, it never launched.

Reply:
Okay but my comment still stands.